pkg/server/status.go

```diff
@@ -17,14 +17,52 @@ limitations under the License.
 package server
 
 import (
-	"errors"
+	"fmt"
 
 	"golang.org/x/net/context"
+	healthapi "google.golang.org/grpc/health/grpc_health_v1"
 
 	runtime "k8s.io/kubernetes/pkg/kubelet/apis/cri/v1alpha1"
 )
 
+const (
+	// runtimeNotReadyReason is the reason reported when runtime is not ready.
+	runtimeNotReadyReason = "ContainerdNotReady"
+	// networkNotReadyReason is the reason reported when network is not ready.
+	networkNotReadyReason = "NetworkPluginNotReady"
+)
+
 // Status returns the status of the runtime.
 func (c *criContainerdService) Status(ctx context.Context, r *runtime.StatusRequest) (*runtime.StatusResponse, error) {
-	return nil, errors.New("not implemented")
+	runtimeCondition := &runtime.RuntimeCondition{
```

Comment:
that which implements a MetadataStore..

SandboxStore, ImageMetadataStore, ...